src/react/utils/index.js

```diff
@@ -1,3 +1,8 @@
+export const defaultMetadataKeys = ['cache-control', 'content-disposition', 'content-encoding',
+    'content-language', 'content-type', 'expires', 'website-redirect-location'];
+
+export const META_WORD = 'x-amz-meta';
```

Comment:
Small thing but I would use `AMZ_META` or similar as it would indicate its from amazon and not one of our own meta values. Plus its 2 words at least. :)